src/pages/StoryForm/StoryComplete.css

```
@@ -0,0 +1,72 @@
+.story-complete-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100dvh;
+  padding: 40px 32px 56px;
+  box-sizing: border-box;
+  background-color: var(--color-background);
+  font-family: "Warhaven", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Warhaven" (font-family-name-quotes)

(font-family-name-quotes)